lib/shelley/src/Cardano/Wallet/Shelley/Network.hs

```diff
@@ -985,75 +841,45 @@ newRewardBalanceFetcher
     -- ^ Used to convert tips for logging
     -> TQueue IO (LocalStateQueryCmd (CardanoBlock StandardCrypto) IO)
     -> IO ( Observer IO W.RewardAccount W.Coin
-          , (AnyCardanoEra, Tip (CardanoBlock StandardCrypto)) -> IO ()
+          , (Maybe AnyCardanoEra, Tip (CardanoBlock StandardCrypto)) -> IO ()
             -- Call on tip-change to refresh
           )
 newRewardBalanceFetcher tr gp queryRewardQ =
     newObserver (contramap MsgObserverLog tr) fetch
   where
     fetch
-        :: (AnyCardanoEra, Tip (CardanoBlock StandardCrypto))
+        :: (Maybe AnyCardanoEra, Tip (CardanoBlock StandardCrypto))
```

Comment:
Hm. This `Tip` is only used for logging. And the log message might lie, since it is not necessarily the tip we're going to use.

Reply:
I removed it from the trace.